### PR TITLE
SCMOD-6182: add last-update-time to job

### PR DESCRIPTION
**Pull request by joseph-lansdowne-opentext**
In the `JobTrackingWorkerIT` tests, I decided not to extend `JobReportingExpectation` to handle the new assertions - hopefully it's clear why from the nature of the assertions.

- ticket: https://autjira.microfocus.com/browse/SCMOD-6182
- build: http://sou-jenkins2.hpeswlab.net/job/JobService/job/JobService~job-service~SCMOD-6182~CI/

### CODE REVIEW

File: job-service-db/src/main/resources/v2.7.0/changelog.xml
Comment (dermot-hardy) [inline job-service-db/src/main/resources/v2.7.0/changelog.xml:144]: Does this not need to come before the procedure updates or does it not matter?

File: job-service/src/main/java/com/hpe/caf/services/job/api/generated/model/Job.java
Comment (dermot-hardy) [inline job-service/src/main/java/com/hpe/caf/services/job/api/generated/model/Job.java]: Some sort of whitespace screw up here.

File: job-service/src/main/java/com/hpe/caf/services/job/api/generated/model/Job.java
Comment (dermot-hardy) [inline job-service/src/main/java/com/hpe/caf/services/job/api/generated/model/Job.java]: And here - re-indent.

**Review by dermot-hardy — APPROVED** [review]
Does adding all new change sets at the bottom of the changelog.xml file mean that we can upgrade from an existing 2.7.0-SNAPSHOT installation?  Or would we only support upgrading from the 2.6.0 release?

**Comment by joseph-lansdowne-opentext** [comment]
I haven't tried running liquibase manually, so I'm not sure.  I assumed it didn't matter, seeing that previous versions' files also had multiple changes to them, and there's no naming convention established for multiple changelogs within a release version.

File: job-service-db/src/main/resources/v2.7.0/changelog.xml
Comment (joseph-lansdowne-opentext) [reply job-service-db/src/main/resources/v2.7.0/changelog.xml:144]: Either way round, there'll be a period of time where the schema and procedure are mismatched, and procedure calls will fail.  As I now understand it, upgrades are expected to have downtime - stop database-using services, upgrade the database, start database-using services - so it seems to me that it shouldn't matter.

**Comment by dermot-hardy** [comment]
I'm not going to worry about it.  We only need to support upgrading from release versions anyway.  Are you going to merge this or do you want me to?  We use the Assignee to indicate who should merge it.  I.e. assign to me if you want me to merge it when PR is approved or assign to yourself if you want review without merging.  No assignee is a bit of a gray area.

**Comment by joseph-lansdowne-opentext** [comment]
Ah, I didn't realise, was just told the committer generally does it.  So yeah, I'll do it.

**Pull request merged**
